taxonomy/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.1.1 on 2020-09-07 09:53
```

Comment:
why are we changing the date here?

Reply:
Date has been changed in rebasing, since I and irfan bhai have migrations with the same name. I have reverted it.
Does it make any difference?

Reply:
No I was just wondering why we did that :) just approved your PR!